### PR TITLE
Adds recent contributors

### PR DESCRIPTION
**Pull request by james-nash**
Added contributors Daniel Wilkinson and Charles 'Ash Blue' Brindle

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,9 @@
     "Marcos Peebles",
     "James Nash (http://cirrus.twiddles.com/)",
     "Valerio Francescangeli",
-    "Daniel Bull"
+    "Daniel Bull",
+    "Daniel Wilkinson",
+    "Charles Brindle (Ash Blue)"
```
Comment (dw-buildit) [inline package.json]: Are brackets reserved for URLs in the `contributors` section of package.json?
Something somewhere might convert Ash Blue to a hyperlink 🤔

Comment (james-nash) [reply package.json]: Good spot. You're right! https://docs.npmjs.com/files/package.json#people-fields-author-contributors

@marcospeebles Perhaps change this to something like `"Charles \"Ash Blue\" Brindle"`?

**Comment by james-nash** [comment]
@marcospeebles We've had some recent contributions from: Gregg Anderson & Allan Corbett.

Perhaps you add them to the list as part of this PR too?

**Pull request merged**